cmd/celestia-appd/cmd/start.go

```diff
@@ -117,6 +118,20 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 				return err
 			}
 
+			switch clientCtx.ChainID {
```

Comment:
if a user specifies no chain ID, what's the default value?

Reply:
This will get logged but no default value will be supplied:

```
				serverCtx.Logger.Info(fmt.Sprintf("No default value exists for the v2 upgrade height when the chainID is %v", clientCtx.ChainID))
```